File: C/输出星期几.c
#include<stdio.h>
#include<stdlib.h>
int main(int argc,char** argv)
{
	FILE *fp;
	if((fp=fopen("result.txt","w"))==NULL)
		return -1;
	int x;
	x=atoi(argv[1]);
	switch(x)
	{
		case 0:fprintf(fp,"S");break;
		case 1:fprintf(fp,"M");break;
		case 2:fprintf(fp,"T");break;
		case 3:fprintf(fp,"W");break;
		case 4:fprintf(fp,"T");break;
		case 5:fprintf(fp,"F");break;
		case 6:fprintf(fp,"S");break;
		default:fprintf(fp,"I");break;
	}	
	fclose(fp);
	return 0;
}

File: Python/输出星期几.py
import sys
x=int(sys.argv[1])
with open('result.txt','w') as fp:
	if x==0 :
		fp.write('S')
	elif x==1 :
		fp.write('M')
	elif x==2 :
		fp.write('T')
	elif x==3 :
		fp.write('W')
	elif x==4 :
		fp.write('T')
	elif x==5 :
		fp.write('F')
	elif x==6 :
		fp.write('S')
	else :
		fp.write('I')